vmm/src/vstate.rs

```diff
@@ -48,17 +49,21 @@ pub enum Error {
     /// The number of configured slots is bigger than the maximum reported by KVM.
     NotEnoughMemorySlots,
     /// Cannot set the local interruption due to bad configuration.
-    LocalIntConfiguration(interrupts::Error),
+    LocalIntConfiguration(arch::interrupts::Error),
     /// Cannot set the memory regions.
     SetUserMemoryRegion(sys_util::Error),
+    #[cfg(target_arch = "x86_64")]
```

Comment:
When we add multiple archs we might want to order errors like these by arch rather than just alphabetically.

Reply:
MSRS setup is available for x86 and x86_64. You should add both here and to all the other errors as well.